docs/dashboard/customize.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "Customize the standard dashboard"
+description: "Netdata's preconfigured dashboard offers many customization options, such as choosing when charts are updated, your preferred theme, and custom text to document processes, and more."
+type: how-to
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/dashboard/customize.mdx
+---
+
+# Customize the standard dashboard
+
+While the [Netdata dashboard](/docs/dashboard/how-dashboard-works.mdx) comes preconfigured with hundreds of charts and
+thousands of metrics, you may want to alter your experience based on a particular use case or preferences.
+
+## Dashboard settings
+
+To change dashboard settings, click the on the **settings** icon ![Import
+icon](https://raw.githubusercontent.com/netdata/netdata-ui/98e31799c1ec0983f433537ff16d2ac2b0d994aa/src/components/icon/assets/gear.svg)
+in the top panel.
+
+These settings only affect how the dashboard behaves in your browser. They take effect immediately and are permanently
+saved to browser local storage (except the refresh on focus / always option). Some settings are applied immediately, and
+others are only reflected after the dashboard is refreshed, which happens automatically.
+
+Here are a few popular settings:
+
+### Change chart legend position
+
+Find this setting under the **Visual** tab. By default, Netdata places the [legend of
+dimensions](/docs/dashboards/charts-dimensions-contexts-families.mdx#dimensions) _below_ charts. Click this toggle to
+move the legend to the _right_ of charts.
+
+### Change theme
+
+Find this setting under the **Visual** tab. Choose between Dark (the default) and White.
+
+## Customize the standard dashboard
```

Comment:
[we have the same section in the web/gui](https://github.com/netdata/netdata/tree/master/web/gui#customizing-the-local-dashboard)

Do you plan to remove it?

> The last push on this front will be the new web server reference, which is mostly done!

I read the comment, but I am not sure you are referencing to the web/gui.

Reply:
Yes, the next PR will add a new reference file and then remove a number of files that are now duplicates, like `/web/gui/README.md`.

Reply:
Thanks for the clarification!